src/model/model.nit

```diff
@@ -570,7 +570,7 @@ class MClass
 	# Is `self` and abstract class?
 	var is_abstract: Bool is lazy do return kind == abstract_kind
 
-	redef fun mdoc_or_fallback do return intro.mdoc_or_fallback
+	redef fun mdoc_or_fallback do return intro.mdoc
```

Comment:
I just do not get why `console_model.nit` did these refinements. Calling `mdoc_or_fallback` seems more robust since it let the fallback propagate.

The introduction do not seems to indicate something https://github.com/nitlang/nit/commit/22bf31681e18cd8be2110b29d75a34970040952d#diff-875ba16ac166e0710a3bb0b1560aeacdR176
@Morriar do you remember something about that?